### PR TITLE
fix terraform validate region requirement

### PR DESCRIPTION
**Pull request by smiller171**

### CODE REVIEW

**Review by cdaniluk — APPROVED** [review]
lgtm

**Comment by smiller171** [comment]
Why does terraform_docs keep behaving differently on my local machine and in gh-actions?

incredibly frustrating

**Comment by smiller171** [comment]
and my auto-commit didn't run because of the failure before it. Need to figure out how to fix that

**Comment by smiller171** [comment]
oh this time the docs was partly my fault...

fixed it so it won't happen again.

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -15,7 +16,29 @@ repos:
...
+              DIRS+=($(dirname "$FILE"))
+            done
+            for DIR in $(printf "%s\n" "${DIRS[@]}" | sort -u)
+            do
+              cd $(dirname "$FILE")
+              terraform init --backend=false
+              terraform validate .
+            done
+          '
+        # entry: bash -c 'echo "$@"'
```
Comment (sblack4) [inline .pre-commit-config.yaml]:
```suggestion
```
extra comment

**Review by sblack4 — APPROVED** [review]
:shipit: it looks good to me but I wonder when we want to start creating new hooks/updating existing ones rather than hacking at the existing ones

**Comment by smiller171** [comment]
> :shipit: it looks good to me but I wonder when we want to start creating new hooks/updating existing ones rather than hacking at the existing ones

Well `terraform validate` seems like a pretty basic one to me.

**Review by cdaniluk — APPROVED** [review]

**Pull request merged**